### DISCUSSION TITLE
Dataviz app using streamlit

### DISCUSSION THREAD

**Discussion by BoBeauf** [f1db/f1db#47] [Show and tell]
Hello everyone,

I think a lot of people have already done what I'm about to show you, but I couldn't find anything in the discussions of the repo.

I made a streamlit app using the data from this repo to do some dataviz and find some insights. The goal is not to do micro-analysis (statistics on the actual seasons). It's more about macro analysis: comparing seasons, analysing team performances over the years, comparing the ladder with different points systems (motoGP, indycar...)...

The app is in French for now (but I will translate it soon).

Have a look around and don't hesitate to let me know if you have any ideas to add.

[https://formula-one-analysis.streamlit.app/](https://formula-one-analysis.streamlit.app/)

**Comment by marceloverdijk** [comment]
Very cool @BoBeauf !
My French is not so good, so I'm looking forward to the English version ;-)

**Comment by BoBeauf** [comment]
Just updated to english !
